contracts/DssVestTopUp.sol

```diff
@@ -159,17 +154,6 @@ contract DssVestTopUp is Ownable {
         return IERC20(paymentToken).balanceOf(address(this));
     }
 
-    /**
-     * @notice Calculates the minimum balance required to keep the upkeep active
-     * @dev Adds a premium on top of the minimum balance to prevent upkeep from going inactive
-     * @return threshold for triggering top up
-     */
-    function getUpkeepThreshold() public view returns (uint256) {
-        uint256 minBalance = keeperRegistry.getMinBalanceForUpkeep(upkeepId);
-        uint256 premium = (minBalance * minBalancePremium) / 100;
-        return minBalance + premium;
-    }
```

Comment:
So this is removed in favour of having the public getter of `threshold`?

Reply:
Correct